language/compiler/ir_to_bytecode/syntax/src/ast.rs

```diff
@@ -289,9 +289,9 @@ pub enum Builtin {
     /// Check if there is a struct object (`StructName` resolved by current module) associated with
     /// the given address
     Exists(StructName, Vec<Type>),
-    /// Get the struct object (`StructName` resolved by current module) associated with the given
-    /// address
-    BorrowGlobal(StructName, Vec<Type>),
+    /// Get a reference to the resource(`StructName` resolved by current module) associated
+    /// with the given address
+    BorrowGlobal(bool, StructName, Vec<Type>),
```

Comment:
Expand the comment to indicate what the bool is for?

Reply:
Will do